docs/topics/jvm/jvm-get-started.md

```diff
@@ -32,26 +32,27 @@ To get started, first download and install the latest version of [IntelliJ IDEA]
 7. Enable the **Add sample code** option to create a file with a sample `"Hello World!"` application.
 8. Click **Create**.
 
-## Create an application
-
-1. If you chose the Gradle build system, you have `build.gradle(.kts)`, a build script file, in your project. It includes
-   the `kotlin("jvm")` plugin and dependencies required for your console application. Ensure that you use the latest
-   version of the plugin:
+  > If you chose the Gradle build system, you have in your project a build script file: `build.gradle(.kts)`. It includes 
```

Comment:
That's an actual step of the instruction. If the user has an old Kotlin version, the readln() example below won't work

Reply:
It seems to me that this step should be at least in the section "Create a project". It relates more to setting up the project than doing something with the application itself. 

And I didn't want it to be the 9th step in the "Create a project" section because in this instruction in the step number 5 we ask a user to choose the IntelliJ build system. For me, it seems strange to do one more separate step "If you chose the Gradle build system". So that's why I think the note format is ok: the information stays here for authors who decide to deviate from instructions.

Reply:
I don't think it relates to Create a project, as it's not a part of the wizard. If you want to keep it as a note, maybe you can place it at the bottom of Create an application, I just think it's weird to have a huge instruction tip at the beginning

Reply:
> maybe you can place it at the bottom of Create an application
I suppose this place is "too late" for a reader.

@koshachy WDYT?

Was the first item here (Create an application) https://kotlinlang.org/docs/jvm-get-started.html#create-an-application
<img width="1151" alt="Screenshot 2022-10-25 at 14 51 31" src="https://user-images.githubusercontent.com/78360457/197777802-1daa29e0-dfb9-47cc-badd-4a97701349f6.png">

Now is the last item (tip) here (Create a project):
<img width="917" alt="Screenshot 2022-10-25 at 14 51 48" src="https://user-images.githubusercontent.com/78360457/197777873-19126277-a99d-4e04-afd2-f9b1cb062ba9.png">

Reply:
I think that "Create a project" is a better place to put this — it's a **result** of this part of the tutorial.

But this note doesn't look good to me. But we could keep this that way.